public/js/icinga/behavior/navigation.js

```diff
@@ -354,6 +360,40 @@
         $(window).trigger('resize');
     };
 
+    /**
+     * Toggle config flyout visibility
+     *
+     * @param {Object} e Event
+     */
+    Navigation.prototype.toggleConfigFlyout = function(e) {
+        var _this = e.data.self;
+        if ($('#layout').is('.config-flyout-open')) {
+            _this.hideConfigFlyout(e);
+        } else {
+            _this.showConfigFlyout(e);
+        }
+    }
+
+    /**
+     * Hide config flyout
+     *
+     * @param {Object} e Event
+     */
+    Navigation.prototype.hideConfigFlyout = function(e) {
+        $('#layout').removeClass('config-flyout-open');
+        $(e.target).closest('.container').removeAttr('data-suspend-autorefresh')
```

Comment:
`delete ...ainer')[0].dataset.suspendAutorefresh;` (Don't forget the semicolon)

Reply:
✅